docs/index.md

```diff
@@ -139,6 +122,27 @@ database for your session.
 There are a number of other options you can pass to the `init` command;
 see the help message for more details.
 
+### An important note on separating tests and production code
+
+Cosmic Ray has a relatively simple view of how to mutate modules. Fundamentally,
+it will attempt to mutate any and all code in a module. This means that if you
+have test code in the same module as your code under test, Cosmic Ray will
+happily mutate the test code along with the production code. This is probably
+not what you want.
+
+The best way to avoid this problem is to keep your test code in separate modules
+from your production code. This way you can tell Cosmic Ray precisely what to
+mutate.
+
+Ideally, your test code will be in a different package form your production
+test. This way you can tell Cosmic Ray to mutate an entire package without
```

Comment:
Missed this one. I believe it should be `production code` not `production test`.